projects/step-core/src/lib/client/generated/models/TableSettings.ts

```diff
@@ -0,0 +1,9 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+import type { ColumnSelection } from './ColumnSelection';
+
+export type TableSettings = {
+  columns: Array<ColumnSelection>;
+};
```

Comment:
@david-stephan 
Potential conflict with new custom columns feature:
https://github.com/exense/step-frontend/pull/782/files#diff-448a2c6edb664aca6f8684c829c82443fc8374a12a366c7f5021b2078c7b5f14

Please check BE changes for the current ticket

Reply:
@dvladir I will change its name